### ISSUE TITLE
Failed to build app after Xcode update to 14.3

### ISSUE THREAD

**Issue by fpollakowsky** [leancodepl/patrol#1245]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/leancodepl/patrol/issues)

### Steps to reproduce

I've use the following command:
```
patrol test \
\--target integration_test/app_test.dart \
\--dart-define=ENVIRONMENT=dev
```

### Actual results

App build failed.

### Logs

<details>
<summary>Logs</summary>

```
Every test target will be run 1 time(s)
• Building app with entrypoint app_test.dart for iOS simulator (debug)...
        2023-05-04 14:26:18.842 xcodebuild[86481:1447494] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
        2023-05-04 14:26:21.049 xcodebuild[86481:1447494] [MT] IDEFileReferenceDebug: [Load] <IDESwiftPackageCore.IDESwiftPackageSpecialFolderFileReference, 0x6000037b6900: name:Documentation.docc path:group:Documentation.docc> Failed to load container at path: /Users/fpollakowsky/StudioProjects/test/build/ios_integ/SourcePackages/checkouts/DefaultsWrapper/Sources/DefaultsWrapper/Documentation.docc, Error: Error Domain=com.apple.dt.IDEContainerErrorDomain Code=6 "Cannot open "Documentation.docc" as a "Swift Package Folder" because it is already open as a "Folder"." UserInfo={NSLocalizedDescription=Cannot open "Documentation.docc" as a "Swift Package Folder" because it is already open as a "Folder".}
        2023-05-04 14:26:26.279 xcodebuild[86481:1447494] Writing error result bundle to /var/folders/2c/fygq9xv55xxf_15bxz5gpkq89jvsyx/T/ResultBundle_2023-04-05_14-26-0026.xcresult
        xcodebuild: error: Failed writing xctestrun file: The folder “Runner_iphonesimulator16.4.xctestrun” doesn’t exist..
✗ Failed to build app with entrypoint app_test.dart for iOS simulator (xcodebuild exited with code 70) (26.0s)
Error: xcodebuild exited with code 70
See the logs above to learn what happened. Also consider running with --verbose. If the logs still aren't useful, then it's a bug - please report it.
 FAIL  app_test.dart on iPhone SE (3rd generation)
```

</details>

### Patrol version

patrol: ^1.1.0
patrol_cli v1.1.7

### Patrol Doctor output

<details>
<summary>Patrol Doctor output</summary>

```
Patrol CLI version: 1.1.7
Program adb not found 
Env var $ANDROID_HOME is not set
Program xcodebuild found in /usr/bin/xcodebuild
Program ideviceinstaller found in /usr/local/bin/ideviceinstaller
Program ios-deploy found in /usr/local/bin/ios-deploy
```

</details>

### Flutter Doctor output

<details>
<summary>Flutter Doctor output</summary>

```
Doctor summary (to see all details, run flutter doctor -v):
[!] Flutter (Channel stable, 3.7.12, on macOS 13.1 22C65 darwin-x64, locale de-DE)
    ! Warning: `dart` on your path resolves to /usr/local/Cellar/dart/2.18.4/libexec/bin/dart, which is not inside your current Flutter SDK checkout at
      /Users/fpollakowsky/fvm/versions/stable. Consider adding /Users/fpollakowsky/fvm/versions/stable/bin to the front of your path.
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.2)
[✓] Connected device (3 available)
[✓] HTTP Host Availability

! Doctor found issues in 1 category.
```

</details>

**Comment by bartekpacia** [comment]
Please run `patrol test` with the  `--verbose` flag, as the issue template says.

**Comment by fpollakowsky** [comment]
Here is the output with the` --verbose` flag:

```
fpollakowsky@macbook17 test-app % /bin/zsh /Users/fpollakowsky/StudioProjects/test-app/execute_integration.sh
An error was encountered processing the command (domain=com.apple.CoreSimulator.SimError, code=405):
Unable to boot device in current state: Booted
Verbose mode enabled. More logs will be printed.

Update available! 1.1.7 → 1.1.9
Run patrol update to update

Received 1 test target(s)
Received test target: /Users/fpollakowsky/StudioProjects/test-app/integration_test/app_test.dart
$ flutter --no-version-check devices --machine
No device specified, using the first one (iPhone SE (3rd generation))
Received 1 device(s) to run on
Received device: iPhone SE (3rd generation)
Every test target will be run 1 time(s)
Received 4 --dart-define(s) (2 custom, 2 internal)
Received custom --dart-define: ENVIRONMENT
Received custom --dart-define: BACKEND_URL
Received internal --dart-define: PATROL_WAIT=0
Received internal --dart-define: INTEGRATION_TEST_SHOULD_REPORT_RESULTS_TO_NATIVE=false
• Building app with entrypoint app_test.dart for iOS simulator (debug)...
$ flutter build ios --no-version-check --config-only --no-codesign --debug --simulator --target /Users/fpollakowsky/StudioProjects/test-app/integration_test/app_test.dart --dart-define ENVIRONMENT=dev --dart-define BACKEND_URL=https%3A%2F%2Fnode.homemuh.synology.me%2FlmwV2 --dart-define PATROL_WAIT=0 --dart-define INTEGRATION_TEST_SHOULD_REPORT_RESULTS_TO_NATIVE=false --dart-define PATROL_TEST_LABEL=app_test.dart
        Building de.test for simulator (ios)...
$ xcodebuild build-for-testing -workspace Runner.xcworkspace -scheme Runner -configuration Debug -sdk iphonesimulator -destination generic/platform=iOS Simulator -quiet -derivedDataPath ../build/ios_integ OTHER_SWIFT_FLAGS=$(inherited) -D PATROL_ENABLED
        2023-05-10 10:10:51.751 xcodebuild[48625:611393] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
        2023-05-10 10:10:52.675 xcodebuild[48625:611393] [MT] IDEFileReferenceDebug: [Load] <IDESwiftPackageCore.IDESwiftPackageSpecialFolderFileReference, 0x600003f3fc00: name:Documentation.docc path:group:Documentation.docc> Failed to load container at path: /Users/fpollakowsky/StudioProjects/test-app/build/ios_integ/SourcePackages/checkouts/DefaultsWrapper/Sources/DefaultsWrapper/Documentation.docc, Error: Error Domain=com.apple.dt.IDEContainerErrorDomain Code=6 "Cannot open "Documentation.docc" as a "Swift Package Folder" because it is already open as a "Folder"." UserInfo={NSLocalizedDescription=Cannot open "Documentation.docc" as a "Swift Package Folder" because it is already open as a "Folder".}
        note: Building targets in dependency order
        error: Unexpected duplicate tasks
            note: Target 'Runner' (project 'Runner') has copy command from '/Users/fpollakowsky/StudioProjects/test-app/ios/Runner/GoogleService-Info.plist' to '/Users/fpollakowsky/StudioProjects/test-app/build/ios_integ/Build/Products/Debug-iphonesimulator/Runner.app/GoogleService-Info.plist'
            note: Target 'Runner' (project 'Runner') has copy command from '/Users/fpollakowsky/StudioProjects/test-app/ios/Runner/GoogleService-Info.plist' to '/Users/fpollakowsky/StudioProjects/test-app/build/ios_integ/Build/Products/Debug-iphonesimulator/Runner.app/GoogleService-Info.plist'
        note: Run script build phase 'Thin Binary' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'Runner' from project 'Runner')
        note: Run script build phase 'Run Script' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'Runner' from project 'Runner')
        2023-05-10 10:11:00.140 xcodebuild[48625:611393] Writing error result bundle to /var/folders/2c/fygq9xv55xxf_15bxz5gpkq89jvsyx/T/ResultBundle_2023-10-05_10-11-0000.xcresult
        xcodebuild: error: Failed writing xctestrun file: The folder “Runner_iphonesimulator16.4.xctestrun” doesn’t exist..
✗ Failed to build app with entrypoint app_test.dart for iOS simulator (xcodebuild exited with code 70) (31.8s)
Error: xcodebuild exited with code 70
#0      throwToolExit (package:patrol_cli/src/base/exceptions.dart:7:3)
#1      IOSTestBackend.build.<anonymous closure> (package:patrol_cli/src/ios/ios_test_backend.dart:123:9)
<asynchronous suspension>
#2      DisposeScope.run (package:dispose_scope/src/dispose_scope.dart:44:7)
<asynchronous suspension>
#3      IOSTestBackend.build (package:patrol_cli/src/ios/ios_test_backend.dart:71:5)
<asynchronous suspension>
#4      TestCommand._builderFor.<anonymous closure> (package:patrol_cli/src/commands/test.dart:229:9)
<asynchronous suspension>
#5      _NativeTestRunner.run.runTestsOnDevice (package:patrol_cli/src/test_runner.dart:230:13)
<asynchronous suspension>
#6      Future.wait.<anonymous closure> (dart:async/future.dart:522:21)
<asynchronous suspension>
#7      _NativeTestRunner.run (package:patrol_cli/src/test_runner.dart:258:5)
<asynchronous suspension>
#8      TestCommand.run (package:patrol_cli/src/commands/test.dart:189:21)
<asynchronous suspension>
#9      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#10     PatrolCommandRunner.runCommand (package:patrol_cli/src/runner/patrol_command_runner.dart:311:18)
<asynchronous suspension>
#11     PatrolCommandRunner.run (package:patrol_cli/src/runner/patrol_command_runner.dart:256:18)
<asynchronous suspension>
#12     patrolCommandRunner (package:patrol_cli/src/runner/patrol_command_runner.dart:66:20)
<asynchronous suspension>
#13     main (file:///Users/fpollakowsky/.pub-cache/hosted/pub.dartlang.org/patrol_cli-1.1.7/bin/main.dart:6:20)
<asynchronous suspension>

See the logs above to learn what happened. Also consider running with --verbose. If the logs still aren't useful, then it's a bug - please report it.
 FAIL  app_test.dart on iPhone SE (3rd generation)
```

**Comment by bartekpacia** [comment]
Here's the error:

```
 error: Unexpected duplicate tasks
            note: Target 'Runner' (project 'Runner') has copy command from '/Users/fpollakowsky/StudioProjects/test-app/ios/Runner/GoogleService-Info.plist' to '/Users/fpollakowsky/StudioProjects/test-app/build/ios_integ/Build/Products/Debug-iphonesimulator/Runner.app/GoogleService-Info.plist'
            note: Target 'Runner' (project 'Runner') has copy command from '/Users/fpollakowsky/StudioProjects/test-app/ios/Runner/GoogleService-Info.plist' to '/Users/fpollakowsky/StudioProjects/test-app/build/ios_integ/Build/Products/Debug-iphonesimulator/Runner.app/GoogleService-Info.plist'
```

I don't know what might be causing this - it's the first time I'm seeing it.

**Comment by fpollakowsky** [comment]
I've fixed the error with the `GoogleService-Info.plist` file by re-referencing this file in Xcode. However, the problem still persists.

Here is the new log:
```
/bin/zsh /Users/fpollakowsky/StudioProjects/test-app/execute_integration.sh
fpollakowsky@macbook17 test-app % /bin/zsh /Users/fpollakowsky/StudioProjects/test-app/execute_integration.sh
An error was encountered processing the command (domain=com.apple.CoreSimulator.SimError, code=405):
Unable to boot device in current state: Booted
Verbose mode enabled. More logs will be printed.
Received 1 test target(s)
Received test target: /Users/fpollakowsky/StudioProjects/test-app/integration_test/app_test.dart
$ flutter --no-version-check devices --machine
No device specified, using the first one (iPhone SE (3rd generation))
Received 1 device(s) to run on
Received device: iPhone SE (3rd generation)
Every test target will be run 1 time(s)
Received 4 --dart-define(s) (2 custom, 2 internal)
Received custom --dart-define: ENVIRONMENT
Received custom --dart-define: BACKEND_URL
Received internal --dart-define: PATROL_WAIT=0
Received internal --dart-define: INTEGRATION_TEST_SHOULD_REPORT_RESULTS_TO_NATIVE=false
• Building app with entrypoint app_test.dart for iOS simulator (debug)...
$ flutter build ios --no-version-check --config-only --no-codesign --debug --simulator --target /Users/fpollakowsky/StudioProjects/test-app/integration_test/app_test.dart --dart-define ENVIRONMENT=dev --dart-define BACKEND_URL=https%3A%2F%2Fnode.homemuh.synology.me%2FlmwV2 --dart-define PATROL_WAIT=0 --dart-define INTEGRATION_TEST_SHOULD_REPORT_RESULTS_TO_NATIVE=false --dart-define PATROL_TEST_LABEL=app_test.dart
        Building de.test for simulator (ios)...
$ xcodebuild build-for-testing -workspace Runner.xcworkspace -scheme Runner -configuration Debug -sdk iphonesimulator -destination generic/platform=iOS Simulator -quiet -derivedDataPath ../build/ios_integ OTHER_SWIFT_FLAGS=$(inherited) -D PATROL_ENABLED
        2023-05-10 12:26:11.605 xcodebuild[25010:989228] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
        2023-05-10 12:26:14.061 xcodebuild[25010:989228] [MT] IDEFileReferenceDebug: [Load] <IDESwiftPackageCore.IDESwiftPackageSpecialFolderFileReference, 0x6000009bb700: name:Documentation.docc path:group:Documentation.docc> Failed to load container at path: /Users/fpollakowsky/StudioProjects/test-app/build/ios_integ/SourcePackages/checkouts/DefaultsWrapper/Sources/DefaultsWrapper/Documentation.docc, Error: Error Domain=com.apple.dt.IDEContainerErrorDomain Code=6 "Cannot open "Documentation.docc" as a "Swift Package Folder" because it is already open as a "Folder"." UserInfo={NSLocalizedDescription=Cannot open "Documentation.docc" as a "Swift Package Folder" because it is already open as a "Folder".}
        /Users/fpollakowsky/.pub-cache/hosted/pub.dev/patrol-1.2.0-dev.2/ios/Classes/AutomatorServer/AutomatorServer.swift:9:15: error: type 'AutomatorServer' does not conform to protocol 'Patrol_NativeAutomatorAsyncProvider'
          final class AutomatorServer: Patrol_NativeAutomatorAsyncProvider {
                      ^
        /Users/fpollakowsky/.pub-cache/hosted/pub.dev/patrol-1.2.0-dev.2/ios/Classes/AutomatorServer/AutomatorServer.swift:9:15: note: do you want to add protocol stubs?
          final class AutomatorServer: Patrol_NativeAutomatorAsyncProvider {
                      ^
        note: Building targets in dependency order
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'CNIOBoringSSL' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'CGRPCZlib' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'CNIOAtomics' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'CNIODarwin' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'CNIOLinux' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'CNIOHTTPParser' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'CNIOWindows' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'FMDB' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'nanopb' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target '_NIODataStructures' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftProtobuf' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'PromisesObjC' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'OrderedSet' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'Logging' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'gRPC-Swift' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'GoogleUtilities' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'FirebaseInstallations' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOConcurrencyHelpers' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'GoogleDataTransport' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIO' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOHPACK' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'FirebaseCore' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOSSL' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOTransportServices' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOPosix' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOCore' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'FirebaseCoreInternal' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOEmbedded' from project 'Pods')
        note: Run script build phase 'Run Script' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'Runner' from project 'Runner')
        note: Run script build phase 'Thin Binary' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'Runner' from project 'Runner')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOFoundationCompat' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOExtras' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOTLS' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOHTTP2' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'SwiftNIOHTTP1' from project 'Pods')
        /Users/fpollakowsky/StudioProjects/test-app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.4.99. (in target 'CNIOBoringSSLShims' from project 'Pods')
        ** TEST BUILD FAILED **
        
✗ Failed to build app with entrypoint app_test.dart for iOS simulator (xcodebuild exited with code 65) (37.6s)
Error: xcodebuild exited with code 65
#0      throwToolExit (package:patrol_cli/src/base/exceptions.dart:7:3)
#1      IOSTestBackend.build.<anonymous closure> (package:patrol_cli/src/ios/ios_test_backend.dart:123:9)
<asynchronous suspension>
#2      DisposeScope.run (package:dispose_scope/src/dispose_scope.dart:44:7)
<asynchronous suspension>
#3      IOSTestBackend.build (package:patrol_cli/src/ios/ios_test_backend.dart:71:5)
<asynchronous suspension>
#4      TestCommand._builderFor.<anonymous closure> (package:patrol_cli/src/commands/test.dart:229:9)
<asynchronous suspension>
#5      _NativeTestRunner.run.runTestsOnDevice (package:patrol_cli/src/test_runner.dart:230:13)
<asynchronous suspension>
#6      Future.wait.<anonymous closure> (dart:async/future.dart:522:21)
<asynchronous suspension>
#7      _NativeTestRunner.run (package:patrol_cli/src/test_runner.dart:258:5)
<asynchronous suspension>
#8      TestCommand.run (package:patrol_cli/src/commands/test.dart:189:21)
<asynchronous suspension>
#9      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#10     PatrolCommandRunner.runCommand (package:patrol_cli/src/runner/patrol_command_runner.dart:311:18)
<asynchronous suspension>
#11     PatrolCommandRunner.run (package:patrol_cli/src/runner/patrol_command_runner.dart:255:18)
<asynchronous suspension>
#12     patrolCommandRunner (package:patrol_cli/src/runner/patrol_command_runner.dart:65:20)
<asynchronous suspension>
#13     main (file:///Users/fpollakowsky/.pub-cache/hosted/pub.dartlang.org/patrol_cli-1.1.9/bin/main.dart:6:20)
<asynchronous suspension>

See the logs above to learn what happened. Also consider running with --verbose. If the logs still aren't useful, then it's a bug - please report it.
 FAIL  app_test.dart on iPhone SE (3rd generation)
```

**Comment by bartekpacia** [comment]
I see you're using `patrol: 1.2.0-dev.2` in your debug logs, but in the issue you're saying you're using `patrol: 1.1.0`?

We don't provide any support for prerelease versions, so I'm going to close it.

If you're interested in test bundling (see #1004), support for it on iOS shipped in [1.2.0-dev.3](https://pub.dev/packages/patrol/versions/1.2.0-dev.3/changelog).
